atomic_reactor/plugins/pre_koji.py

```diff
@@ -20,7 +25,7 @@ class KojiPlugin(PreBuildPlugin):
     key = "koji"
     is_allowed_to_fail = False
 
-    def __init__(self, tasker, workflow, target, hub, root):
+    def __init__(self, tasker, workflow, target, hub, root, koji_ssl_certs=None):
```

Comment:
The original plan was 'either reuse kojisecret for cert or download the cert from server - and then set ```sslcacert``` in yum repos). It seems that yum doesn't work this way - but we might revisit this.

I'm gonna clean this up and repush